ubuntudesign/www.canonical.com
# Change www.ubuntu.com references to https

Does what it says on the tin.

Changing links and redirects from http, to https. Ubuntu.com is now https and we are creating an unneeded redirect.

# QA
Please note that redirects do not update till a server restart. Just in case you already have the dev server running.
Check that the links and redirects work/look good!
